cmc-ccd-adapter/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
```

Comment:
not using mapstruct @sabahirfan ?

Reply:
no, as mapstruct does not support immutable objects atm.